src/component/hover.tsx

```diff
@@ -53,18 +58,21 @@ export class Popup extends Widget {
     }
 
     protected onUpdateRequest(msg: Message): void {
+        this.setGeometry();
         this.setGeometry();
         super.onUpdateRequest(msg);
     }
 
     protected onAfterAttach(msg: Message): void {
         document.addEventListener('click', this, false);
         document.addEventListener('keypress', this, false);
+        window.addEventListener('resize', this, false);
```

Comment:
why call `window.addEventListener('resize', this, false);` and then `document.removeEventListener('resize', this, false);`